packages/stacked_tools/CHANGELOG.md

```diff
@@ -1,4 +1,14 @@
+## 0.2.4-beta.4
+
+- Removed lib and test folder from the default paths because there is no need to include them as they can't be changed
```

Comment:
For this change, I want us to provide a warning to the users that has older configs so that they can update their file. SImilar to the comment on the path change that I left. 

We want to have this functionality in the code now so we can know how to make it scalable as we most likely will change other parts of the cli that we want to notify the user about in the future.